manim/__main__.py

```diff
@@ -36,7 +36,10 @@ def open_file_if_needed(file_writer):
 
         for file_path in file_paths:
             if current_os == "Windows":
-                os.startfile(os.path.dirname(file_path))
+                if file_writer_config["preview"]:
+                    os.startfile(file_path)
+                if file_writer_config["show_file_in_finder"]:
```

Comment:
```suggestion
                else:
```
I think this would do.

Reply:
I disagree. If we do that, the user won't be able to both get a preview and show the file in the file explorer.

Reply:
Would be better to be more explicit though, don't you think?